uuid.go

```diff
@@ -284,13 +284,20 @@ func (u *UUID) UnmarshalBinary(data []byte) (err error) {
 
 // Value implements the driver.Valuer interface.
 func (u UUID) Value() (driver.Value, error) {
+	if u == Nil {
+		return nil, nil
+	}
 	return u.String(), nil
 }
 
 // Scan implements the sql.Scanner interface.
 // A 16-byte slice is handled by UnmarshalBinary, while
 // a longer byte slice or a string is handled by UnmarshalText.
 func (u *UUID) Scan(src interface{}) error {
+	if src == nil {
```

Comment:
Rather than a conditional here, just add another case to the type switch below: `case nil:`